### PR TITLE
istioctl x es adopt prometheus annotations

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

`istioctl x es` adopt prometheus annotations when output is `prom-merged`
this will make you can retrieve stats for ztunnel by running:
```console
 out/linux_amd64/istioctl x es -nistio-system ztunnel-wbtp4 -o prom-merged
```

### CODE REVIEW

File: operator/pkg/util/k8s.go [hunk 1]
```diff
@@ -128,3 +130,23 @@ func CreateNamespace(cs kubernetes.Interface, namespace string, network string,
 
 	return nil
 }
+
+func PrometheusPathAndPort(pod *v1.Pod) (string, int, error) {
+	path := "/metrics"
+	port := 9090
```
Comment (hanxiaop) [inline operator/pkg/util/k8s.go:136]: Why also change the default value?

Comment (zirain) [reply operator/pkg/util/k8s.go:136]: prometheus will use it, if you don't specify a specific path via `prometheus.io/path`.

Comment (hanxiaop) [reply operator/pkg/util/k8s.go:136]: I see how it works. Thanks

**Review by hanxiaop — APPROVED** [review]

**Comment by zirain** [comment]
kindly ping @istio/wg-environments-maintainers

**Review by howardjohn — APPROVED** [review]

**Pull request merged**